pacai/ui/pacmanDisplay.py

```diff
@@ -0,0 +1,663 @@
+            return
+
+        if self.gifSkipFrames != 0 and self.frame % self.gifSkipFrames != 0:
+            return
+
+        self.frameImages.append(graphicsUtils.getPostscript())
+
+    def write_gif(self):
+        if not self.gifPath:
+            return
+
+        # Delay the dependency unless someone actually needs it.
+        import imageio
+
+        images = [imageio.imread(io.BytesIO(image.encode())) for image in self.frameImages]
+        imageio.mimwrite(self.gifPath, images, fps = self.gifFPS, subrectangles = True)
+
+    def drawDistributions(self, state):
+        walls = state.getWalls()
+        dist = []
+        for x in range(walls.width):
+            distx = []
+            dist.append(distx)
+            for y in range(walls.height):
+                screen_x, screen_y = self.to_screen((x, y))
+                block = graphicsUtils.square(screen_x, screen_y, 0.5 * self.gridSize,
+                    color = graphicsConstants.BACKGROUND_COLOR, filled = 1, behind = 2)
+                distx.append(block)
+        self.distributionImages = dist
+
+    def drawStaticObjects(self, state):
+        layout = self.layout
+        self.drawWalls(layout.walls)
+        self.food = self.drawFood(layout.food)
+        self.capsules = self.drawCapsules(layout.capsules)
+        graphicsUtils.refresh()
+
+    def drawAgentObjects(self, state):
+        self.agentImages = []  # (agentState, image)
+        for index, agent in enumerate(state.getAgentStates()):
+            if agent.isPacman():
+                image = self.drawPacman(agent, index)
+                self.agentImages.append((agent, image))
+            else:
+                image = self.drawGhost(agent, index)
+                self.agentImages.append((agent, image))
+        graphicsUtils.refresh()
+
+    def swapImages(self, agentIndex, newState):
+        """
+        Changes an image from a ghost to a pacman or vis versa (for capture)
+        """
+        prevState, prevImage = self.agentImages[agentIndex]
+        for item in prevImage:
+            graphicsUtils.remove_from_screen(item)
+
+        if newState.isPacman():
+            image = self.drawPacman(newState, agentIndex)
+            self.agentImages[agentIndex] = (newState, image)
+        else:
+            image = self.drawGhost(newState, agentIndex)
+            self.agentImages[agentIndex] = (newState, image)
+        graphicsUtils.refresh()
+
+    def make_window(self, width, height):
+        grid_width = (width - 1) * self.gridSize
+        grid_height = (height - 1) * self.gridSize
+        screen_width = 2 * self.gridSize + grid_width
+        screen_height = 2 * self.gridSize + grid_height + graphicsConstants.INFO_PANE_HEIGHT
+
+        graphicsUtils.begin_graphics(screen_width, screen_height,
+                graphicsConstants.BACKGROUND_COLOR, "Pacman")
+
+    def drawPacman(self, pacman, index):
+        position = self.getPosition(pacman)
+        screen_point = self.to_screen(position)
```

Comment:
So I still am seeing a bunch of variables that are screen_point instead of screenPoint. I know its kind of a pain to fix but we should try and make them consistent. I am not going to be doing a comment for all of them since there are quite a few.

Reply:
I think I fixed al the ones in pacmanDisplay, captureGraphicsDisplay, and graphicsDisplay. graphicsUtil is written almost entirely in snake case: Should I change everything in there as well?

Reply:
No for now thats fine.